sismic/code/python.py

```diff
@@ -54,43 +20,10 @@ def notify(name, **kwargs):
     return notify
 
 
-class PythonEvaluator(Evaluator):
+class PythonEvaluator(EnhancedEvaluator):
     """
     A code evaluator that understands Python.
 
-    Depending on the method that is called, the context can expose additional values:
```

Comment:
Docstring should remain, notably because it is currently used by Sphinx to generate "official" documentation.

Reply:
One possibility is to explain in the documentation that we have two base evaluators, a very simple one that does nothing, and another one that supports several features (time, active(), send(), ....), and one concrete implementation of that one (EnhancedEvaluator should be an ABCMeta then) that supports Python code.

Reply:
I moved the doc (to EnhancedEvaluator) but wanted to wait for your feedback regarding creating Mixin's before doing anymore with it. If Sphinx will not pick it up where it is, then OK, drop it for now.